lms/djangoapps/grades/course_grade.py

```diff
@@ -294,14 +293,7 @@ def attempted(self):  # lint-amnesty, pylint: disable=invalid-overridden-method
         Returns whether any of the subsections in this course
         have been attempted by the student.
         """
-        if assume_zero_if_absent(self.course_data.course_key):
-            return True
-
-        for chapter in self.chapter_grades.values():
-            for subsection_grade in chapter['sections']:
-                if subsection_grade.all_total.first_attempted:
-                    return True
-        return False
+        return True
```

Comment:
Should we just remove this function entirely if it always returns `True`?

Reply:
This method override the same [method](https://github.com/openedx/edx-platform/pull/30540/files#diff-f0d17d91bec07c249a69bfeb1b4be40078e8c433136af33644f429eeddcab9b3L57) from CourseGradeBase that return `False`. I think we should leave it.